snippets/cdc-acm-console/cdc-acm-console.overlay

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/ {
+	chosen {
+		zephyr,console = &cdc_acm_uart0;
+	};
+};
+
+&zephyr_udc0 {
+	cdc_acm_uart0: cdc_acm_uart0 {
```

Comment:
Like for the shields ["Besides, in order to avoid name conflicts with devices that may be defined at board level, it is advised, specifically for shields devicetree descriptions, to provide a device nodelabel is the form <device>_<shield>, for instance:"](https://docs.zephyrproject.org/latest/hardware/porting/shields.html#shield-porting-and-configuration), we need a namespace for nodelabels in snippets to avoid conflicts.


`snippet_console_cdc_acm_uart: snippet_console_cdc_acm_uart {`

`snippet_shell_cdc_acm_uart: snippet_shell_cdc_acm_uart {` 

?

Reply:
>  we need a namespace for nodelabels in snippets

I'll propose `_snippet_name_with_underscores`, so `cdc-acm-console` has namespace `_snippet_cdc_acm_console`.

Reply:
Hmm, maybe `snippet_name_with_underscores` (so `cdc-acm-console` -> `snippet_cdc_acm_console`) is better actually

Reply:
> Hmm, maybe `snippet_name_with_underscores` (so `cdc-acm-console` -> `snippet_cdc_acm_console`) is better actually

Like `snippet_cdc_acm_console_uart: snippet_cdc_acm_console_uart {` ?